cmd/svcat/output/json.go

```diff
@@ -0,0 +1,23 @@
+package output
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+)
+
+func writeJSON(w io.Writer, obj interface{}, n int) {
+
+	if n == 0 {
```

Comment:
Ignoring the value they set seems a bit unexpected to me. If it's a `*int` then checking null would be less surprising than converting 0 -> 3. Though then it makes it hard to pass in an int... 🤔 

Just something to think about, we can tweak later.

Reply:
For now, I just went ahead and set it to do three spaces. If we run into the need for variable indent on JSON output, let's revisit?